File: ejemplo_api/src/main/java/ni/edu/ejemplo_api/models/Producto.java
package ni.edu.ejemplo_api.models;

/**
 * Project: ejemplo_api
 * From: ni.edu.ejemplo_api.models
 * Created by: Sara Zambrana
 * On: 04/05/2026 at 18:21
 * All rights reserved 2026.
 */
public class Producto {
    private String nombreProducto;
    private Double precio;

    public String getNombreProducto() {
        return nombreProducto;
    }

    public void setNombreProducto(String nombreProducto) {
        this.nombreProducto = nombreProducto;
    }

    public Double getPrecio() {
        return precio;
    }

    public void setPrecio(Double precio) {
        this.precio = precio;
    }

    public Producto(String nombreProducto, Double precio) {
        this.nombreProducto = nombreProducto;
        this.precio = precio;
    }
}

File: Ejercicios_Sesion#03/Ejercicio02-SistemaProductos/src/Producto.kt
/**
 * Project: Ejercicio02-SistemaProductos
 * From:
 * Created by: Sara Zambrana
 * On: 16/03/2026 at 21:25
 * All rights reserved 2026.*/

data class Producto (
    var nombre : String,
    var precio : Double,
    var cantidad : Int
) {
    fun mostrarProducto(){
        println("\nProducto: $nombre")
        println("Precio: $precio")
        println("Cantidad: $cantidad\n")
        println("-----------------------")
    }
}

